CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 - Updated Harvard Cannon operational run scripts to use `huce_cascade` instead of `huce_intel`; also added `sapphire`
+- Changed exponent 'e' to 'd' in gckpp_Rates.F90 to prevent precision error in external models
+
+### Fixed
+- Fixed unit conversions in GEOS-only code
```

Comment:
NOTE: gckpp_Rates.F90 is updated each time we rebuild KPP.  Better to change "e" to "d" in the fullchem.eqn file.